pkg/monitortests/authentication/requiredsccmonitortests/monitortest.go

```diff
@@ -130,18 +157,29 @@ func (w *requiredSCCAnnotationChecker) CollectData(ctx context.Context, storageD
 		}
 
 		failureMsg := strings.Join(failures, "\n")
-		junits = append(junits,
-			&junitapi.JUnitTestCase{
-				Name:          testName,
-				SystemOut:     failureMsg,
-				FailureOutput: &junitapi.FailureOutput{Output: failureMsg},
-			},
-
-			// add a successful test with the same name to cause a flake
-			&junitapi.JUnitTestCase{
-				Name: testName,
-			},
-		)
+
+		if isFlakingNamespace {
+			// report as a flake
+			junits = append(junits,
+				&junitapi.JUnitTestCase{
+					Name:          testName,
+					SystemOut:     failureMsg,
+					FailureOutput: &junitapi.FailureOutput{Output: failureMsg},
+				},
+				&junitapi.JUnitTestCase{
+					Name: testName,
+				},
+			)
+		} else {
+			// enforce as a failure
+			junits = append(junits,
+				&junitapi.JUnitTestCase{
+					Name:          testName,
+					SystemOut:     failureMsg,
+					FailureOutput: &junitapi.FailureOutput{Output: failureMsg},
+				},
+			)
+		}
```

Comment:
In both cases we need one failing `JUnitTestCase`, so append it always. If we need a flake as well, append one.